docs/source/user-guide/tabular/quickstarts/plot_quick_model_evaluation.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
```

Comment:
Currently this still appears both in the Tutorials folder and in the Quickstarts folder...
(unlike the others - it was **added**, instead of **renamed and moved**

Reply:
Sure, I'll make the mentioned changes.